From ded72d71cbcdf31cad3088790195a056b57b47a2 Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Tue, 10 Oct 2017 09:41:17 -0500
Subject: [PATCH 5/5] Set strict variables and basemodulepath in puppet.conf

Also sets ordering = title-hash so that puppet 4 will run the same as puppet 3

Signed-off-by: Robert Church <robert.church@windriver.com>
---
 conf/puppet.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/conf/puppet.conf b/conf/puppet.conf
index 67dba46..4efaf55 100644
--- a/conf/puppet.conf
+++ b/conf/puppet.conf
@@ -4,3 +4,9 @@
 # - https://puppet.com/docs/puppet/latest/config_about_settings.html
 # - https://puppet.com/docs/puppet/latest/config_file_main.html
 # - https://puppet.com/docs/puppet/latest/configuration.html
+[main]
+    # Prevent the use of undefined variables
+    strict_variables = true
+    ordering = title-hash
+    # Set the path to StarlingX puppet modules
+    basemodulepath = /usr/share/puppet/modules:/usr/share/openstack-puppet/modules
-- 
2.16.6

